docs/docs/vite-configuration.md

```diff
@@ -0,0 +1,72 @@
+---
+description: If you have to configure Vite, here's how
+---
+
+# Vite Configuration
+
+Redwood uses Vite. One of Redwood's tenets is convention over configuration.
+
+Vite is an awesome build tool, but we don't want it to be something that you have to be familiar with to be productive.
+So it's worth repeating that you don't have to do any of this, because we configure everything you will need out of the box with a Redwood vite plugin.
```

Comment:
```suggestion
So it's worth repeating that you don't have to do any of this, because we configure everything you will need out of the box with a Redwood Vite plugin.
```